File: mbrs/src/main/java/com/ftn/mbrs/service/RadnoVremeService.java
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::RadnoVreme


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.RadnoVreme;

public interface RadnoVremeService {

	public RadnoVreme save(RadnoVreme radnoVreme, Long stanicaId);
	
	public RadnoVreme update(RadnoVreme radnoVreme);

	public String delete(Long id);
	
	public RadnoVreme findOne(Long id);
	
	public List<RadnoVreme> findAll();
}

File: mbrs/src/main/resources/static/js/services/RadnoVremeService.js
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::RadnoVreme

angular.module('gremoNaElektrikoApp.RadnoVremeService', [])
	.factory('RadnoVremeService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/radnoVreme');
		}
		
		factory.createRadnoVreme = function(radnoVreme , stanicaId) {
			return $http.post('/gremoNaElektriko/radnoVreme'+'/'+stanicaId, radnoVreme);
		}

		factory.deleteRadnoVreme = function(id) {
			return $http.delete('/gremoNaElektriko/radnoVreme/'+id);
		}
		
		factory.updateRadnoVreme = function(radnoVreme) {
			return $http.put('/gremoNaElektriko/radnoVreme', radnoVreme);
		}
		
		return factory;
	});